libsentrykube/iap.py

```diff
@@ -21,45 +16,29 @@
 )
 
 
-def _tcp_port_check(port: int) -> bool:
-    with closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as sock:
-        # The error indicator is 0 if the operation succeeded, otherwise the value of the
-        # errno variable.
-        return sock.connect_ex(("127.0.0.1", port)) == 0
-
-
-def _dns_check() -> None:
-    try:
-        socket.gethostbyname("kubernetes")
-    except socket.gaierror:
-        raise RuntimeError(
-            "Need to make host 'kubernetes' resolved to localhost\n\n"
-            "Action:\n"
-            "- You need to add an entry to /etc/hosts, like\n\n"
-            "127.0.0.1       kubernetes\n"
-        )
-
-
-def _dns_endpoint_check(control_plane_host: str, quiet: bool) -> bool:
+def _is_external_connection_allowed(
```

Comment:
Do we need this check at all, or can we just assume DNS connections are enabled?